command/src/com/mirth/connect/cli/CommandLineInterface.java

```diff
@@ -179,10 +180,21 @@ private void runShell(String server, String user, String password, String script
             client = new Client(server);
             this.debug = debug;
 
-            LoginStatus loginStatus = client.login(user, password);
-
-            if (loginStatus.getStatus() != LoginStatus.Status.SUCCESS) {
-                error("Could not login to server.", null);
+            LoginStatus loginStatus;

```

Comment:
```suggestion
            LoginStatus loginStatus = null;
```

Changes below will complain that loginStatus may have not been initialized without this.